# A message file to display basic information about a recorded bag file

# The list of topics subscribed to
string[] topic_list

# A full absolute file path
string location	

# The starting time of the bag based on the ros Time API
time start_time

# Ending time of the bag (same convention)
time end_time

